### PR TITLE
Align lint with containerd

### PR DESCRIPTION
**Pull request by dmcgowan**
Adds some stricter linting with fewer file exceptions

### CODE REVIEW

File: test/verify.sh [hunk 1]
```diff
@@ -26,4 +26,4 @@ cd "${ROOT}"
 
 make install.tools
 make verify
-GOOS=windows make verify
+#GOOS=windows make verify
```
Comment (dims) [inline test/verify.sh:29]: Did this `#` sneak in or we need this @dmcgowan ?

Comment (mikebrow) [reply test/verify.sh:29]: if we pull from here we may want to add it back in as a separate GOOS=windows ./test/verify.sh call on the pull-cri-containerd-verify ... I don't remember exactly why verify was moved from travis to prow way back when. There may be some dependency at the sig-node test grid for verify testing on windows..

Comment (mikebrow) [reply test/verify.sh:29]: containerd runs verify for each GOOS as a separate bucket.. and I agree we should go that route.  So I'm good with making this change we just need to open a PR to add windows verify on prow.

**Comment by dmcgowan** [comment]
Added windows back, just disabled the integration and ignoring `*.pb.go` files

**Review by mikebrow — APPROVED** [review]
LGTM

**Comment by dims** [comment]
LGTM

thanks @dmcgowan

File: test/verify.sh (hunk 1 above)
Comment (mikebrow) [reply test/verify.sh]: fair ^. I think typically the Kubernetes guys like to have verify in pre-submit before running e2e buckets.. may be why they wanted a prow pre-submit. Course we run github actions.. and I don't think we can force that as a pre-submit before the prow bot picks up our `/test` requests.  We should consider if it's ok to run verify twice for this reason..

**Pull request merged**